cmd/helm/install.go

```diff
@@ -278,27 +278,37 @@ func (i *installCmd) run() error {
 		helm.InstallDisableCRDHook(i.disableCRDHook),
 		helm.InstallTimeout(i.timeout),
 		helm.InstallWait(i.wait))
-	if err != nil {
-		return prettyError(err)
+
+	// If there is an error while waiting, make a call without waiting to get the release content
```

Comment:
should we be wrapping this in a `if settings.Debug` much like what was done in `helm rollback` and `helm upgrade`?